src/cmdc/datasets/bea/data.py

```diff
@@ -0,0 +1,140 @@
+#%% setup
+from concurrent.futures import ThreadPoolExecutor
+import textwrap
+import requests
+import pandas as pd
+
+from cmdc.datasets.base import OnConflictNothingBase
+
+
+def _make_bea_request(**kw):
+    key = "94DEC0DC-E6F5-4DC9-8E78-64D61AA34433"
```

Comment:
We probably don't want this key to be public info.

Reply:
Yeah... I'll change it to read from env var